### ISSUE TITLE
Update color of pending URL text in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10866]
## Description
When a new node is waiting to be published (or a node that doesn't have a corresponding FE page), there is a pending URL within Drupal CMS.  This URL is disabled and is displayed in light gray text, which does not meet WCAG contrast ratio requirements. It is recommended that, the text be updated so that is does meet these requirements and is easier for all users to see.

There are 2 colors that this could be updated to: `#595959` or `#757576`

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/192299947-4a46d875-27eb-4a9a-877f-68bee569041b.png)
Example URL: https://training-qtpbfivdfzrlrlyiqwmanznaobpek1ca.demo.cms.va.gov/banner/homepage-banner

## Acceptance Criteria
- [ ] Consult design
- [ ] Update pending URL text to approved color of `#595959`
- [ ] Confirm with Accessibility Lead

**Comment by laflannery** [comment]
@edmund-dunn color has been approved by Blake and added to ACs, we will be using `#595959`
